async_test.go

```diff
@@ -315,7 +330,7 @@ func TestAsyncTimeout(t *testing.T) {
 	assert.Equal(t, *message, *readMessage)
 	assert.Nil(t, content)
 
-	time.Sleep(defaultDeadline * 10)
+	time.Sleep(defaultDeadline * 5)
```

Comment:
is there a reason why we changed this? Just improve test times?

Reply:
Yep